pkg/apis/core/v1alpha1/defaults_test.go

```diff
@@ -167,6 +167,71 @@ var _ = Describe("Defaults", func() {
 			})
 		})
 
+		Describe("Worker Swap", func() {
+			It("should default the swap behaviour for a worker pool", func() {
+				falseVar := false
+				obj.Spec.Provider.Workers = []Worker{
+					{
+						Kubernetes: &WorkerKubernetes{
+							Kubelet: &KubeletConfig{},
+						},
+					},
+				}
+				obj.Spec.Provider.Workers[0].Kubernetes.Kubelet.FailSwapOn = &falseVar
```

Comment:
```suggestion
				obj.Spec.Provider.Workers[0].Kubernetes.Kubelet.FailSwapOn = pointer.Bool(false)
```
and other occurences